net/relayd/src/etc/inc/plugins.inc.d/relayd.inc

```diff
@@ -1,533 +1,80 @@
 <?php
 
-/*
-    Copyright (C) 2016 Franco Fichtner <franco@opnsense.org>
```

Comment:
please leave this copyright intact, it was from the "relayd.inc" refactoring. the code is still there, even though it was reformatted / changed in syntax.